content/ja/handbook/finance/jihu-support/jihu-contribution-process.md

```
@@ -0,0 +1,128 @@
+JiHu チームからのコントリビューションは、JiHu 独自の変更があるかどうかに応じて2つの方法で行われます。
+
+- アップストリーム方式 - GitLab Inc. リポジトリへのマージリクエストから開始する。
+- 独自仕様とアップストリーム混在方式 - すべての変更を GitLab JiHu プロジェクトに対するマージリクエストで開始し、独自仕様以外の変更をアップストリームプロジェクトに対するマージリクエストで行う。
+
+JiHu からのコントリビューションを識別するため、JiHu チームからのすべてのアップストリームコントリビューションには `~"JiHu contribution"` ラベルが自動的に付与されます。ラベルが正確に適用されるよう、`gitlab-jh` チームは [`gitlab-jh/jh-team`](https://gitlab.com/groups/gitlab-jh/jh-team/-/group_members?with_inherited_permissions=exclude) のダイレクトメンバーを現在のチームメンバーで常に最新の状態に保つ必要があります。
+
+JiHu enablement の効率性エージングとレビュー指標は、この[ダッシュボード](https://app.periscopedata.com/shared/f7af38e3-0cea-4219-a5d3-6f6b25f10244?)で公開されています。
+
+[エンジニアリング生産性](/handbook/engineering/infrastructure-platforms/developer-experience/)チームは JiHu エンジニアリング enablement 効率ツールと指標の DRI です。
+
+### アップストリームコントリビューションのガイドライン
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)